lints/lint_ub_common_name_invalid.go

```diff
@@ -0,0 +1,44 @@
+// lint_ub_common_name_invalid.go
+/************************************************
+RFC 5280: A.1
+	* In this Appendix, there is a list of upperbounds
+	for fields in a x509 Certificate. *
+	ub-common-name INTEGER ::= 64
+************************************************/
+
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type ubCommonNameInvalid struct {
+	// Internal data here
+}
+
+func (l *ubCommonNameInvalid) Initialize() error {
+	return nil
+}
+
+func (l *ubCommonNameInvalid) CheckApplies(c *x509.Certificate) bool {
+	// Add conditions for application here
+	return true
+}
+
+func (l *ubCommonNameInvalid) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	if len(c.Subject.CommonName) <= 64 {
+		return ResultStruct{Result: Pass}, nil
+	} else {
+		return ResultStruct{Result: Error}, nil
+	}
+}
+
+func init() {
+	RegisterLint(&Lint{
+		Name:          "e_ub_common_name_invalid",
+		Description:   "The 'Common Name' field must be less than 64 integers long.",
```

Comment:
@jddicki2 for now, make this explicit. We'll need to figure out where all these fields apply carefully in future upper bound tests and make sure we're not missing anything